File: Groove.SP/Groove.SP.Core/Entities/NoteModel.cs

namespace Groove.SP.Core.Entities
{
    public class NoteModel : Entity
    {
        #region Properties

        public long Id { get; set; }

        public string GlobalObjectId { get; set; }

        public string Category { get; set; }

        public string NoteText { get; set; }

        public string ExtendedData { get; set; }

        //Name of user (in The Portal database) who created note
        public string Owner { get; set; }


        #endregion
    }
}

File: Groove.SP/groove-sp-web/src/app/core/models/note.model.ts

/** Base model data for note */
export class NoteModel {
    id?: number;
    category?: string;
    noteText?: string;
    extendedData?: string;
    createdDate?: Date;
    /**Email of user who created*/
    createdBy?: string;
    /**Name of user who created*/
    owner?: string;

    constructor(owner?: string, createdBy?: string, createdDate?: Date) {
        this.owner = owner;
        this.createdBy = createdBy;
        this.createdDate = createdDate;
    }
}
